gui/include/gui/options.h

```diff
@@ -451,6 +451,9 @@ class options : private Uncopyable,
   wxRadioButton *pCBCourseUp, *pCBNorthUp, *pRBSizeAuto, *pRBSizeManual;
   int k_tides;
 
+  // For the OpenGL Options
+  wxButton *bOpenGL;
```

Comment:
The new variable bOpenGL is  not  required and actually confusing. The reason is that the button is created with a unique is ID_OPENGLOPTIONS. This means that is it easy get a hold of the button using WxWindow::FindWindowById(ID_OPENGLOPTIONS). 

Furthermore, to have both a class variable and an unique ID is indeed confusing